### DISCUSSION TITLE
2021/06/08/azure-devops-terraform-pipeline-with-checkov-approvals/

### DISCUSSION THREAD

**Discussion by giscus** [russmckendrick/blog#4] [Comment]
# 2021/06/08/azure-devops-terraform-pipeline-with-checkov-approvals/

It’s been just over a year since I first posted about the Azure DevOps Terraform Pipeline I used to use, I say used to use, because that pipeline is now a little outdated. This posts covers the updated Pipeline I am starting to deploy along side my Terraform code.
Pipeline Overview The pipeline itself has expanded a little and it now not only uses stages but also depending what Terraform is planning on doing it will trigger a manual approval process should there be any resources being destroyed.

https://www.russ.cloud/2021/06/08/azure-devops-terraform-pipeline-with-checkov-approvals/

**Comment by russmckendrick** [comment]
This is just a test to see if the comment system works, nothing to see here, please move on - unless you want to comment on this post, then please do. :grinning:

**Comment by ahmetsyildirim** [comment]
Just wanted to say what a great article! Came across in the middle of my research regarding DevOps approaches adapted for IaC projects, and this has been a great finding for me.

**Comment by rafaelvelosoAZ** [comment]
Nice article thanks for share, only one thing that I was thiking about, how can I execute the Checkov scan only on my terraform.tfplan? If there is any way.

**Reply by schosterbarak** [reply]
@rafaelvelosoAZ here is an example of checkov and tfplan: https://github.com/bridgecrewio/checkov/blob/0348ed4fc7d4fe4e9e49f3c339c7981be1580cc0/docs/4.Integrations/Terraform%20Scanning.md

**Comment by donnyachmadi** [comment]
Great article, thank you so much for sharing this. I was looking to implement the Checkov on the Pipeline, and this guide answered my question. Really appreciate :)

**Reply by russmckendrick** [reply]
Thanks, glad you found it useful
